iris_core/db.js

```diff
@@ -114,18 +114,17 @@ iris.dbPopulate = function () {
 
   // Delete any existing schema so they can be re-written
```

Comment:
Do we need to delete the indexes here as well or are they automatically re-initialised afterwards?